tests/installation/start_install_via_yui.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+# Summary: This test module performs the necessary actions
+#          to start the installation process.
+
+# - click on install button in 'Installation settings'
+# - handle confirm installation popup
+# - click on install
+
+use strict;
+use warnings;
+use parent "installbasetest";
+
+sub run {
+    my $overview_controller = $testapi::distri->get_overview_controller();
+    $overview_controller->get_overview_page()->press_install();
```

Comment:
we should interact mainly with the controller in this layer, except exceptions, so we will need here just:
`installation_settings->install()`